### PR TITLE
Apple sign in

### LINKED ISSUE #694: Add Apple Authentication

### PR DESCRIPTION
**Pull request by confraria**
closes #694

### CODE REVIEW

**Comment by confraria** [comment]
Added #704 as a separate issue. This one is ready review. Still need to align the apple logo on sign-in but it's just replacing the bitmap.

**Comment by confraria** [comment]
Simulator Screen Shot - iPhone 8 - 2022-09-19 at 14 27 46

**Comment by confraria** [comment]
## How to run on real ios device
- Install expo-cli globally `npm i -g expo-cli` 
- Run in the simulator first `npm run ios` (don't try to login, apple id does not work on simulator)
- [Setup code signing certificates in xcode for developement](https://github.com/expo/fyi/blob/main/setup-xcode-signing.md) (reach out to me at this point)
- Install [expo go](https://apps.apple.com/us/app/expo-go/id982107779) on your device
- Connect the ios device to your mac and select trust computer when prompted.
- run `expo run:ios --device --no-build-cache`
- Select your device from the list

**Comment by simoneb** [comment]
> ## How to run on real ios device
> * Install expo-cli globally `npm i -g expo-cli`
> * Run in the simulator first `npm run ios` (don't try to login, apple id does not work on simulator)
> * [Setup code signing certificates in xcode for developement](https://github.com/expo/fyi/blob/main/setup-xcode-signing.md) (reach out to me at this point)
> * Install [expo go](https://apps.apple.com/us/app/expo-go/id982107779) on your device
> * Connect the ios device to your mac and select trust computer when prompted.
> * run `expo run:ios --device --no-build-cache`
> * Select your device from the list

are these general development instructions? if so, maybe it's worth including them in the readme

**Comment by confraria** [comment]
> are these general development instructions? if so, maybe it's worth including them in the readme

 I'll add them to the README after checking the steps worked fine with @radomird

**Comment by confraria** [comment]
> > are these general development instructions? if so, maybe it's worth including them in the readme
> 
> I'll add them to the README after checking the steps worked fine with @radomird

added to the README ✅

File: README.md [hunk 1]
```diff
@@ -26,6 +26,20 @@ In order to successfully run the Optic-expo app locally you will need the follow
 1. Scan the QR code on your terminal or go to `exp://172.22.22.56:19000`
 1. Once the app loads, if you get the signin screen, in your terminal run `expo login -u <username> -p <password>` and reload the app
 
+## Running on a iOS device
+
+1. Create an issue in this repo asking for apple connect developer access for the app
+1. Install expo-cli globally npm i -g expo-cli
```
Comment (radomird) [inline README.md:32]:
```suggestion
1. Install expo-cli globally `npm i -g expo-cli`
```

File: README.md [hunk 2]
```diff
@@ -26,6 +26,20 @@ In order to successfully run the Optic-expo app locally you will need the follow
...
+1. Run in the simulator first npm run ios
```
Comment (radomird) [inline README.md:33]:
```suggestion
1. Run in the simulator first `npm run ios`
```

**Pull request merged**